src/frontend/next/src/pages/LogoIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import Image from 'next/image';
+
+const Logo = require('../images/logo.webp');
+
+// Explicitly state that LogoIcon can only accept numbers as a type when creating a Logo
+const LogoIcon = ({ height, width }: { height: number; width: number }) => {
+  return (
+    <>
+      <Image src={Logo} alt="Telescope Logo" height={height} width={width} />
+    </>
+  );
+};
+
```

Comment:
Maybe create an interface for props type and call it LogoIconProps
```
import {FC} from 'react';
const interface LogoIconProps {
    height: number,
    width: number,
}
const LogoIcon: FC<LogoIconProps> = ({height, width}) => {
    .....
}
```

Reply:
FC stands for functional component that would take in props in the form of LogoIconProps

Reply:
Kk on it!